### PR TITLE
Slight change to build, and better documentation

### PR DESCRIPTION
**Pull request by standage**
More thorough documentation of the steps required for a rebuild, as discussed in https://github.com/bioforensics/MicroHapDB/pull/87#pullrequestreview-894216472.

### CODE REVIEW

**Comment by standage** [comment]
Hey @RyanBerger98, please take a look at this and let me know if it's clear. If so, go ahead and merge it into master and then into your sun2020 branch, and then follow the instructions for the second pass (you've already done the first).

**Comment by standage** [comment]
cc @rnmitchell and @danejo3, this is relevant to your pending PRs.

**Comment by RyanBerger98** [comment]
Other than my comment above this update is clear and looks good to me.

**Comment by standage** [comment]
@RyanBerger98 I think I missed the comment. 😬

**Comment by RyanBerger98** [comment]
> @RyanBerger98 I think I missed the comment. 😬

My comment was:
>You mentioned also having to run get-marker-rsids.py in your comment on my merge request. Would that be need to be ran in this set of commands as well?

**Review by RyanBerger98 — APPROVED** [review]
As per our conversation, this looks good to me. Will merge soon.

File: dbbuild/README.md [hunk 1]
```diff
@@ -46,6 +46,25 @@ If the Snakemake build process completes successfully, copy the newly created da
...
+```
+cd sources/1kgp/
+rm frequency.tsv marker-informativeness.tsv marker-aes.tsv marker-fst.tsv marker-rsids-MicroHapDB-latest.tsv
+snakemake --cores 1 -p all
+cd ../../
+rm marker.tsv frequency.tsv
+snakemake --configfile config.json --cores 1 -p tables
+cp *.tsv sources/1kgp/marker-aes.tsv ../microhapdb/data/
+```
+
```
[the remark this answers is not in the document]
Comment (standage) [reply dbbuild/README.md:64]: Just seeing this now, lol. But good question. To clarify, after deleting the `marker-rsids-MicroHapDB-latest.tsv` file, the updated Snakemake for 1kgp will automatically re-run this script now.

File: dbbuild/sources/1kgp/Snakefile [hunk 2]
```diff
@@ -38,18 +38,18 @@ rule aes:
 
 rule markerrsids:
     input: 'get-marker-rsids.py'
-    output: 'marker-rsids-MicroHapDB-0.7.tsv'
-    shell: './get-marker-rsids.py > marker-rsids-MicroHapDB-0.7.tsv'
+    output: 'marker-rsids-MicroHapDB-latest.tsv'
+    shell: './get-marker-rsids.py > marker-rsids-MicroHapDB-latest.tsv'
```
Comment (standage) [inline dbbuild/sources/1kgp/Snakefile:42]: Badabing badaboom.

**Pull request merged**